Sadot/User/OrderManagement/OrderForm.cs

```diff
@@ -289,14 +295,14 @@ public void FillCurrentTabButtons(string type, TabPage currTab)
         private void FillListWithCancels()
         {
             int lastRowIndex = linesInExistOrders.Count();
-            if(order.isCancels())
+            if (order.isCancels())
             {
                 cancelsInOrder = new List<CancellationsInOrder>(db.GetCancelsOfOrder(order.OrderID));
                 for (int i = 0; i < cancelsInOrder.Count; i++)
                 {
                     dgvOrderList.Rows.Add(cancelsInOrder[i].printName(), 1, cancelsInOrder[i].PriceToSub * -1);
                     dgvOrderList.Rows[lastRowIndex + i].DefaultCellStyle.BackColor = Color.Yellow;
-                }  
+                }
```

Comment:
Move back

Reply:
Didn't understand. what do you mean?

Reply:
The "}" has to much spaces from the start